components/authz-service/server/v2/project_update_manager.go

```diff
@@ -41,6 +41,7 @@ const (
 var ProjectUpdateDomainServices = []string{
 	"ingest",
 	"compliance",
+	"nodemanager",
```

Comment:
Adding the nodemanager to one of the services that get called for project updates.